keyboards/mitosis/keymaps/via/keymap.c

```diff
@@ -0,0 +1,129 @@
+// This is the VIA Configurator compatibility layout, derived from carvac_dv.
+
+#include QMK_KEYBOARD_H
+
+enum mitosis_layers
+{
+    _STD,
+    _NUM,
+    _FN,
+    _EXTRA
+};
+
+
+//Mousekeys
+#define MOUSEKEY_DELAY 300
+#define MOUSEKEY_INTERNAL 50
+#define MOUSEKEY_MAX_SPEED 20
+#define MOUSEKEY_TIME_TO_MAX 30
+#define MOUSEKEY_WHEEL_MAX_SPEED 8
+#define MOUSEKEY_WHEEL_TIME_TO_MAX 40
```

Comment:
These do not belong here, and must be in the `config.h` file for the keymap.  Otherwise, they won't get picked up. 
```suggestion
```